awx/main/models/workflow.py

```diff
@@ -405,6 +411,12 @@ def node_prompts_rejected(self):
                 node_list.append(node.pk)
         return node_list
 
+    '''
+    RelatedJobsMixin
+    '''
+    def _get_active_jobs(self):
+        return WorkflowJob.objects.filter(status__in=ACTIVE_STATES)
```

Comment:
both this and the JobTemplate filter don't filter to the template.

Reply:
whoops